src/metalookup/core/metadata_manager.py

```diff
@@ -82,16 +83,12 @@ async def create_extractor(extractor: Type[Extractor]) -> Extractor:
     async def extract(self, message: Input, extra: bool) -> Output:
         """
         Call the different registered extractors concurrently with given input.
-        :param message: The message holding URL and or HAR (splash response) to analyse. If no splash response
-                        is contained the splash container will be queried before invoking the extractors.
+        :param message: The message holding URL to analyse.
```

Comment:
If the message contains only the URL - why not hand over the url directly?

Reply:
Yes, turning this into a GET with empty body would also be more REST like imho. However, then we somehow need to URLencode the query parameter which would make testing a little less comfortable (eventually?).
Not in the scope of this PR thou:P

Reply:
:D same thought, nice!